core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java

```diff
@@ -884,7 +885,7 @@ private final void parseTransaction(XMLExtendedStreamReader reader, Configuratio
             }
             case TRANSACTION_PROTOCOL: {
                if (reader.getSchema().since(11, 0)) {
-                  throw ParseUtils.unexpectedAttribute(reader, attribute.getLocalName());
+                  throw ParseUtils.attributeRemoved(reader, i, null);
```

Comment:
instead of passing null as a parameter, could we use overloading in ParseUtils ?